src/screens/NotificationScreen.js

```diff
@@ -1,23 +1,172 @@
+      <View>
+        {notification
+          .filter((item) => {
+            return item.status === "OVERDUE";
+          })
+          .map((item, index) => {
+            return content(item, index);
+          })}
+      </View>
+    );
+  };
+
+  const statusToday = () => {
+    return (
+      <View>
+        {notification
+          .filter((item) => {
+            return item.status === "TODAY";
+          })
+          .map((item, index) => {
+            return content(item, index);
+          })}
+      </View>
+    );
+  };
+
+  const content = (item, index) => {
+    if (item.date === null) {
+      return noSchedule();
+    } else {
+      return (
+        <View key={index}>
+          <ContentItem
+            name={item.name}
+            vaccineName={item.vaccineName}
+            date={item.date}
+          />
+        </View>
+      );
+    }
+  };
+
+  const noSchedule = () => {
+    return (
+      <View style={styles.noStatusContainer}>
+        <Image
+          style={styles.image}
+          source={require("../../assets/icons/baby_smile.png")}
+        />
+        <Text style={[styles.noStatus, styles.grey]}>TAKE A REST!</Text>
+      </View>
+    );
+  };
+
   return (
-    <SafeAreaView style={styles.container}>
-      <Text style={styles.text}>This is notification screen</Text>
-    </SafeAreaView>
+    <ScrollView style={screenConfig.container}>
+      <View style={styles.textContainer}>
+        <View style={styles.container}>
+          <Text style={[styles.status, styles.red]}>{"TODAY"}</Text>
+          {statusToday()}
+        </View>
+      </View>
+      <View style={styles.textContainer}>
+        <View style={styles.container}>
+          <Text style={[styles.status, styles.blue]}>{"UPCOMING"}</Text>
+          {statusUpcoming()}
+        </View>
+      </View>
+      <View style={styles.textContainer}>
+        <View style={styles.container}>
+          <Text style={[styles.status, styles.blue]}>{"OVERDUE"}</Text>
+          {statusOverdue()}
+        </View>
+      </View>
+    </ScrollView>
   );
 };
 
 const styles = StyleSheet.create({
   container: {
-    backgroundColor: colors.white,
-    flex: 1,
+    marginLeft: scale(15),
+    marginTop: verticalScale(15),
+    marginBottom: verticalScale(15),
+  },
+  textContainer: {
+    backgroundColor: colorConfig.white,
+    width: scale(349),
+    marginTop: verticalScale(10),
+  },
+  status: {
+    fontFamily: "fredokaOne-regular",
+    fontSize: verticalScale(12),
+  },
+  red: {
+    color: colorConfig.red,
+  },
+  blue: {
+    color: colorConfig.blue,
+  },
+  grey: {
+    color: colorConfig.grey,
+  },
+  image: {
+    width: scale(50),
+    height: verticalScale(60),
+  },
+  noStatusContainer: {
+    flexDirection: "row",
+    marginTop: verticalScale(5),
+    alignItems: "center",
+    alignSelf: "center",
   },
-  text: {
-    textAlign: "center",
+  noStatus: {
+    fontFamily: "fredokaOne-regular",
+    fontSize: verticalScale(12),
```

Comment:
Make a font size bigger for no status text. When you check Hena's design, you can see the font size is slightly bigger than the section font size